packages/flutter_tools/test/general.shard/asset_bundle_test.dart

```diff
@@ -8,44 +8,42 @@ import 'package:file/file.dart';
 import 'package:file/memory.dart';
 import 'package:flutter_tools/src/asset.dart';
 import 'package:flutter_tools/src/base/file_system.dart';
+import 'package:flutter_tools/src/base/logger.dart';
 import 'package:flutter_tools/src/base/platform.dart';
 import 'package:flutter_tools/src/bundle.dart';
-import 'package:flutter_tools/src/cache.dart';
 import 'package:flutter_tools/src/devfs.dart';
-import 'package:flutter_tools/src/globals.dart' as globals;
 import 'package:mockito/mockito.dart';
 
 import '../src/common.dart';
 import '../src/context.dart';
 
+
+final Platform platform = FakePlatform(
```

Comment:
Updated all tests to use a consistent memory file system and linux platform.

Reply:
do we not want to test other platforms?

Reply:
I don't think we need to (I could be convinced otherwise), because ultimately these are unit tests of the asset logic which already abstracts out the platform through the Uri and FileSystem interface